File: GraphQLServerJavaInesMR/src/main/java/com/example/graphqlserverjavainesmr/data/modelo/PersonajeEntity.java
package com.example.graphqlserverjavainesmr.data.modelo;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Getter
@Setter
@Builder
@ToString
@NoArgsConstructor
@Table(name = "personajes")
@AllArgsConstructor
public class PersonajeEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private int id;
    @Column(name = "nombre", nullable = false)
    private String nombre;
    @Column(name = "descripcion", nullable = false)
    private String descripcion;
    @ManyToOne
    @JoinColumn(name = "id_videojuego", nullable = false)
    private VideojuegoEntity videojuego;

}

File: GraphQLServerInesMR/src/main/kotlin/com/example/graphqlserverinesmr/data/modelo/PersonajeEntity.kt
package com.example.graphqlserverinesmr.data.modelo

import jakarta.persistence.*
import lombok.*

@Entity
@Getter
@Setter
@Builder
@ToString
@NoArgsConstructor
@Table(name = "personajes")
@AllArgsConstructor
class PersonajeEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    val id: Int = 0
    val nombre: String = ""
    val descripcion: String = ""

}
